core/src/main/scala/org/scalacheck/ops/ScalaCheckImplicits.scala

```diff
@@ -1,21 +1,25 @@
 package org.scalacheck.ops
 
 import org.scalacheck.Gen
-import org.scalacheck.ops.time.{ImplicitJavaTimeGenerators, JavaTimeImplicits}
+import org.scalacheck.ops.time.{ImplicitJavaTimeGenerators, JavaTimeImplicits, TruncatedJavaTimeImplicits}
 
 import scala.language.implicitConversions
 import scala.reflect.ClassTag
 
 trait ScalaCheckImplicits
   extends ArbitraryAsGen
+  with ImplicitGenFromConfig
   with JavaTimeImplicits
+  with TruncatedJavaTimeImplicits
   with ImplicitJavaTimeGenerators
   with ShrinkLargeTuples {
 
   implicit def genToGenOps[T](gen: Gen[T]): GenOps[T] = new GenOps(gen)
 
   implicit def genObjectToGenOps(gen: Gen.type): GenOps.type = GenOps
 
-  implicit def genToGenOrThrow[T: ClassTag](generator: Gen[T]): GenOrThrow[T] = new GenOrThrow[T](generator)
+  // No longer an implicit, but leaving this here for binary compatibility
+  @deprecated("Use implicit conversion to GenFromConfig instead.", "2.7.0")
+  def genToGenOrThrow[T: ClassTag](generator: Gen[T]): GenOrThrow[T] = new GenOrThrow[T](generator)
```

Comment:
This code change makes the old `GenOrThrow` extension methods no longer implicitly available, but instead, the `ImplicitGenFromConfig` implicits will apply the new `GenFromConfig` extension methods which look mostly the same (with deprecation warnings) to provide source code compatibility. This will be removed entirely in the next major release.